person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative 'nameable'
+
```

Comment:
- [ ] @jfoyarzo great work on your decoration however you forgot to import the capitalize_decorator and trimmer_decorator into your person.rb file so that you can use the classes to decorate your person class